lighthouse-core/test/gather/computed/first-interactive-test.js

```diff
@@ -28,7 +28,7 @@ describe('FirstInteractive computed artifact:', () => {
     return computedArtifacts.requestFirstInteractive({traceEvents: tooShortTrace}).then(() => {
       assert.ok(false, 'should have thrown for short trace');
     }).catch(err => {
-      assert.equal(err.message, 'trace not at least 5 seconds longer than FMP');
+      assert.ok(err.message, 'FMP_TOO_LATE_FOR_FCPUI');
```

Comment:
i was expecting `FMP_TOO_LATE_FOR_FCPUI` to be the err.code, and the long string as the .message.  What am i missing?

Reply:
Maybe some wires got crossed. I expected the long string to just be used for display and not the one used in the stack/Sentry/CLI/etc.

 i.e. I want to see 
```
Error: FMP_TOO_LATE_FOR_FCPUI
    at repl:1:5
    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
```

and not 
```
Error: Your page took too long to load. Please follow the opportunities in the report to reduce your page load time, and then try re-running Lighthouse.
    at repl:1:5
    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
```

Reply:
Yeah that makes sense. Agree that the friendlyMessage is too wordy to be what we see in stack traces.  Hmmm... 
How about a basic `LHError: ${err.code}` ? Just prefixing the string with our custom error type.

Reply:
Oh I like it,

```js
LHError: FMP_TOO_LATE_FOR_FCPUI
    at repl:1:5
    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
```

right?

or are you suggesting

```js
Error: LHError: FMP_TOO_LATE_FOR_FCPUI
    at repl:1:5
    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
```

Reply:
1st one: `LHError: FMP_TOO_LATE_FOR_FCPUI`. 
A la "TypeError" etc.